ESSArch_TA/config/settings.py

```diff
@@ -211,6 +211,8 @@
 DATABASES = {'default': dj_database_url.parse(url=DATABASE_URL)}
 
 # Cache
+REDIS_CLIENT_CLASS = os.environ.get('REDIS_CLIENT_CLASS', 'django_redis.client.DefaultClient')
```

Comment:
This is never used

Reply:
WOT! Lazy copy-paste of me! Will fix! Great catch!